testsuite/forge/src/backend/k8s/chaos.rs

```diff
@@ -285,4 +287,23 @@ impl K8sSwarm {
             INVALID_NODE_STRING
         }
     }
+
+    fn get_service_name(&self, node: &AccountAddress) -> Option<String> {
+        if let Some(validator) = self.validator(*node) {
+            validator.service_name()
+        } else if let Some(fullnode) = self.full_node(*node) {
+            fullnode.service_name()
+        } else {
+            // TODO: should we throw an error here instead of failing silently?
+            None
+        }
+    }
+
+    pub(crate) fn get_service_targets(&self, target_nodes: &[AccountAddress]) -> String {
+        target_nodes
+            .iter()
+            .filter_map(|node| self.get_service_name(node))
```

Comment:
Shouldn't this be returning the ip addresses instead of the service name?

Reply:
Services names get resolved via DNS. so still works.